apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -103,7 +103,7 @@ class AnimationPickerListItem extends React.Component {
             />
           )}
         </div>
-        <div style={labelStyle}>{this.props.label}</div>
+        {this.props.label && <div style={labelStyle}>{this.props.label}</div>}
```

Comment:
(this isn't the right line to comment on, but github won't let me comment in the proper place)

let's add some vertical padding (probably just paddingBottom) to animations that don't have a label, just to give some more visual separation between them

Reply:
Thoughts? This is 15 px of padding below the animations.

![Screenshot 2020-06-04 at 2 24 31 PM](https://user-images.githubusercontent.com/46464143/83811752-435d7380-a66f-11ea-9a12-14ae20a4654d.png)

Reply:
For context, I think there's about ~20px of space between the icons if there's a label but that looked a bit much when I tried that.

Reply:
yeah, that looks great. thank you!